src/plugin-slots/CourseAuthoringUnitSidebarSlot/README.v1.md

```diff
@@ -0,0 +1,86 @@
+# CourseAuthoringUnitSidebarSlot
```

Comment:
It'd be nice to note that this is an old version of the slot that is rendered conditionally based on `isUnitVerticalType`, doesn't wrap `SplitTestSidebarInfo`, and any other notable differences - and that using the v2 version is recommended.

Reply:
@brian-smith-tcril I have updated the v1 docs to add a note about upgrading to V2 and listed the shortcomings of v1 in practical terms as well.